psalm.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<psalm
+	errorBaseline="tests/psalm-baseline.xml"
```

Comment:
this line and the `tests/psalm-baseline.xml` file can be dropped as well. Same with the `vendor-bin/require` directory.